CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to `sky` will be documented in this file
 
+## 2.1.4 - to be releasd
```

Comment:
you can revert these changes in this file since it is automatically generated when making a new release.
and add this to the description of the PR

Reply:
Ok, will do.